src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -1292,7 +1292,14 @@
     <!-- BinderOpt is added as a temporary solution for IOperation implementation and should probably be removed in the future -->
     <Field Name="BinderOpt" Type="Binder" Null="allow"/>
   </Node>
-  
+
+  <Node Name="UnboundObjectCreationExpression" Base="BoundExpression">
```

Comment:
Since the object initializer is logically part of the (un)bound object creation expression, it should be captured here. Probably as a SyntaxNode. The caller should not assume that the syntax of this bound node is of the proper form, as that will prevent us from using it as an intermediary in translations (of perhaps new, different syntax forms) later.

Reply:
Done. 

Please review the follow-up PR at https://github.com/dotnet/roslyn/pull/29167 after this goes in. thanks.